### DISCUSSION TITLE
Separated touchscreen osu!mania zone

### DISCUSSION THREAD

**Discussion by Oliveoil144** [ppy/osu#17737] [Ideas] [answered]
It would be more comfortable to have that feature coming up to osu!lazer mobile for future updates!
https://youtu.be/Rh7YwwAigyM

**Comment by frenzibyte** [comment, marked as answer]
I don't understand this request, can you elaborate further? Are you perhaps looking for #4201?

In fact, it's already there but needs further adjustments to actually be playable.

**Reply by peppy** [reply]
I think #4201 definitely covers this.
